brayns/common/camera/AbstractManipulator.h

```diff
@@ -31,7 +31,15 @@ namespace brayns
 class AbstractManipulator
 {
 public:
-    AbstractManipulator(Camera& camera, KeyboardHandler& keyboardHandler);
+
+    enum AxisMode
```

Comment:
enum class, lower-case names for values

Reply:
What do mean ? AxisMode -> axisMode ? or GLOBAL_Y -> globalY ?

Reply:
Is AxisMode a value?

   enum class AxisMode
   {
       globalY

Reply:
Done